app/scripts/views/NodeViews/Base.js

```diff
@@ -46,18 +43,22 @@ define(['backbone', 'jqueryuidraggable', 'bootstrap', 'Hammer'], function(Backbo
       this.model.on('evalFailed', this.onEvalFailed, this );
       this.model.on('evalBegin', this.onEvalBegin, this );
 
-      this.makeDraggable();
```

Comment:
this method uses much workspace information so I moved it into WorkspaceView which will call it right after creation of node view